.gitignore

```diff
@@ -48,3 +48,25 @@ passphrase
 .coverage.*
 coverage.xml
 htmlcov/
+
+\.vscode/settings\.json
+
+\.pytest_cache/v/cache/
+
+\.vscode/launch\.json
+
+lib/cherrypy/test/access\.log
+
+lib/cherrypy/test/error\.log
+
+lib/cherrypy/test/test\.access\.log
+
+lib/cherrypy/test/test\.error\.log
+
+Pipfile
```

Comment:
It looks like this gitignore includes comments above sections describing what they are, it would be better to follow that example I think. Also you shouldn't need to escape `.`